src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int numberBall = new Random().nextInt(100);
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
lets make them class-level variables - to avoid creating a new instance each time this method is invoked

make a constant for 100 - avoid magic numbers